pkg/addon/utils.go

```diff
@@ -442,7 +442,7 @@ func generateAnnotation(meta *Meta) map[string]string {
 }
 
 func isErrorCueRenderPathNotFound(err error, path string) bool {
-	return err.Error() == fmt.Sprintf("var(path=%s) not exist", path)
+	return err.Error() == fmt.Sprintf("failed to lookup value: var(path=%s) not exist", path)
```

Comment:
this kind of compare is dangerous, I think use errros.As or strings.Contain can be better?